vector-im/element-android
# Fix description of verified sessions (PSG-991)

## Type of change


## Content
We needed to create new string key to deprecate translations of other languages.



## Motivation and context



## Screenshots / GIFs





## Tests



- Enable new session management from Labs settings
- Navigate to Settings > Security & Privacy > Show All Sessions
- Click View Details button of the verified current session section
- Click "Learn more" and see the new description in bottom sheet
- The same result is applied to session details of a verified session in other sessions section

## Tested devices

- OS version(s):

## Checklist
